Add `textmode.js` (browser-based textmode / ASCII visuals)

humanbydefinition opened the pull request:
This PR adds **textmode.js** to the *Libraries* section.

textmode.js is a browser-based creative coding library for real-time textmode / ASCII graphics. It is not a live-coding–specific system, but its real-time rendering model makes it well suited for live-coded visual work.

In addition to live coding, textmode.js can be used for generative art, long-form visual sketches, and interactive works responding to mouse, keyboard, and touch input.

The library is also available inside the collaborative live coding environment [flok.cc](https://flok.cc), where it can be used in live coding contexts, such as rendering Hydra canvases through a textmode frame and creating audio-reactive visuals using audio information from Strudel panes.

More information and documentation: https://code.textmode.art/

humanbydefinition committed:
Add textmode.js to the list of creative coding libraries

humanbydefinition committed:
Update description of textmode.js in README.md for clarity and focus on use cases

humanbydefinition committed:
Add synth.textmode.art to the list of live coding environments and update textmode.js description

lvm commented:
Hi there! this looks awesome but could you move libs to the libraries section ? ty!

humanbydefinition committed:
Merge remote-tracking branch 'upstream/master'

humanbydefinition committed:
Move textmode.js from Languages to Libraries and tools section

humanbydefinition committed:
Add textmode.synth.js to Libraries and tools section

humanbydefinition commented:
> Hi there! this looks awesome but could you move libs to the libraries section ? ty!

Hey hey, thanks for chiming in! Updated the PR, and also added another live-coding centered library that has been published since.

https://github.com/humanbydefinition/textmode.synth.js

It is an add-on library for `textmode.js` and a derivative work of `hydra-synth`, with a very similar syntax. It's also available within the `synth.textmode.art` live coding environment.

lvm commented:
excellent! thank you
